### PR TITLE
WIP: 6-Translation-Mechanism

### PR DESCRIPTION
**Pull request by jhculb**
WIP: no work

### CODE REVIEW

File: src/rml-reified/nested-1.ttl [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        rr:predicate rdf:type ;
+        rml:objectMap [
+            rr:reference rdf:Statement
+        ]
+    ];
+
+    rr:predicateObjectMap [
+        rr:predicate rdf:subject ;
+        rml:objectMap [
+            rml:quotedTriplesMap :firstTM
```
Comment (dachafra) [inline src/rml-reified/nested-1.ttl:67]: this is not correct!

Comment (dachafra) [reply src/rml-reified/nested-1.ttl:67]: You cannot use `quotedTriplesMap` in RML reified mappings, as they are plan RML (you have to follow the standard spec https://rml.io/specs/rml/)

**Review by dachafra — CHANGES_REQUESTED** [review]
Take into account the two main comments from the other PR:

1. For RML-reified you CANNOT use `rml:quotedTriplesMap` as they are plain are mappings (you have to follow the standard spec https://rml.io/specs/rml/)
2. You have to take a look at RML-star spec to understand how `rml:NonAssertTriplesMap` works as there are TriplesMap that will need this annotation (https://w3id.org/kg-construct/rml-star)

File: src/rml-reified/nested-1.ttl (hunk 1 above)
Comment (jhculb) [reply src/rml-reified/nested-1.ttl]: These changes should have been merged in from branch 4 and 5

**Review by dachafra — APPROVED** [review]
As I think here we only have some syntax problems, they should be merged

**Comment by jhculb** [comment]
> As I think here we only have some syntax problems, they should be merged

Hi @dachafra, this branch was intended to focus on developing the translation mechanism (so this would be the primary development branch until the end of the challenge). I was hoping to do a final merge to main once I had everything sorted and working. 

Which syntax errors have you spotted?

**Comment by dachafra** [comment]
Maybe I was a bit confused with some comments from yesterday that said to us to review in 6, forget it ;-)

**Comment by jhculb** [comment]
So I believe that tasks 3 and 5 were agreed complete on this list, could you check 4 is also complete please? :)

- [ ] 3 RDF reification statements
- [ ] 4 RML-star mappings
- [ ] 5 RML reification statements
- [ ] 6 Translation mechanism

I am working on the translation mechanism with the assumption that tasks 3, 4 & 5 are complete until further notice.

**Comment by jhculb** [comment]
Another case of github merging two branches by accident

**Pull request merged**